src/digitalmarketplace/components/footer/__snapshots__/template.test.js.snap

```diff
@@ -95,6 +95,11 @@ exports[`footer renders a footer component with all our standard links 1`] = `
                     Privacy notice
                   </a>
                 </li>
+                <li class=\\"govuk-footer__inline-list-item\\">
+                  <a class=\\"govuk-footer__link\\" href=\\"http://www.github.com\\">
```

Comment:
Needs updating to live URL!

Reply:
I think that might be out of scope for this - this testing snapshot looks like it's using dummy URLs for all of these footer links.